mantidproject/mantid
# Allow PlotPeakByLogValue to accept a minimizer string

Fixes [#11651](http://trac.mantidproject.org/mantid/ticket/11651).

To test:
- Review documentation changes and unit tests
- Do a fit of multiple spectra using FABADA

There is a script [here](https://gist.github.com/DanNixon/4a0cf0c3e80318be5dbf) that does a fit, notice the minimizer property and try adding other workspace outputs (see [docs](http://docs.mantidproject.org/nightly/concepts/FABADA.html#fabada-specific-outputs)), see that they are output and grouped in the same way parameter tables, etc are.

If the data is not in the standard test data then you can get it from `Babylon5/Public/DanNixon`.
